front/src/common-app/components/appbar/appbar.styles.ts

```diff
@@ -0,0 +1,58 @@
+import { css } from 'emotion';
+import { theme } from 'core/theme';
+const { palette, typography, spacing } = theme;
+
+export const appbarContainer = css`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  height: ${spacing(80)};
+  background-image: linear-gradient(
+    60deg,
+    rgb(255, 87, 51) 250px,
+    #d9d900 250px,
+    white 90%
+  );
+  border-bottom: 2px solid rgb(255, 87, 51);
+`;
+
+export const logo = css`
+  margin-top: ${spacing(5)};
+  height: ${spacing(60)};
+  fill: ${palette.text.primary};
+  margin-left: 2.4rem;
+`;
+
+export const navList = css`
+  display: flex;
+  margin-left: auto;
+  list-style: none;
+`;
+
+export const listItem = css`
+  margin-right: 1.5rem;
+  overflow: hidden;
+
+  &:last-of-type {
+    margin-right: 2.4rem;
```

Comment:
Not sure if we should calculate this based on the standard Theme spacing measure adding @Nasdan to the loop

Reply:
Yes it's. You could use `theme.spacing(<number>)` where number could be `1` or `1.5` or `-2.445`...

The rem units are similar to me, but it depends on font-size's html element